includes/data-stores/class-wc-data-store-wp.php

```diff
@@ -555,9 +555,10 @@ protected function get_search_stopwords() {
 	 * @since 3.6.0
 	 * @param int    $id ID of object to update.
 	 * @param string $table Lookup table name.
+	 * @param bool   $ignore_manage_stock If set to true, stock quantity will be returned regardless of _manage_stock meta value on the product.
 	 * @return array
 	 */
-	protected function get_data_for_lookup_table( $id, $table ) {
+	protected function get_data_for_lookup_table( $id, $table, $ignore_manage_stock = false ) {
```

Comment:
It feels wrong to add `$ignore_manage_stock` here as this parameter only makes sense in the context of products. Should we maybe create a more generic parameter like `array $options` or maybe we can get rid of this parameter and always ignore the value of manage_stock and update the stock quantity?

Reply:
Perhaps you're right, I was thinking about adding a more general attribute, but then changed it back as lots of times I saw that planning for future that never comes just introduces more unnecessary abstractions. However, I went with your other suggestion, seems like ignoring the db value of manage_stock should be fine. Updated in 68a3252.